pages/landing/hipaa/template/contact/_hero.html

```diff
@@ -1,8 +1,8 @@
 <div class="container">
   <div class="row">
     <div class="col-xs-12 text-center">
-      <h1>Join the waitlist</h1>
-      <p style="margin-bottom:4px" class="lead">The HIPAA Reference Architecture product is currently in invite-only mode!<br>Please join our waitlist to be notified when it is available.</p>
+      <h1>Request early access</h1>
+      <p style="margin-bottom:4px" class="lead">The HIPAA Reference Architecture product is currently in invite-only mode!<br>Please let us know that you're interested.</p>
```

Comment:
Not sure how important consistency is, but in the Heroku landing page, we've updated the text to:

> This product is currently in an invite-only private beta.
> Use this form to request access and we'll notify you when you can get it!

Reply:
Updated in 83985751.